setup.py

```diff
@@ -10,61 +11,65 @@
 
 
 def find_scripts(script_dir, prefix):
-    script_list = [os.path.join(script_dir, f) for f in os.listdir(script_dir) if f.startswith(prefix)]
+    script_list = [f for f in Path(script_dir).iterdir() if f.name.startswith(prefix)]
```

Comment:
This comprehension list will be a list of `pathlib.Path` objects and not a list of stings. Is this intended ? I assume it does not matter a lot, but I'd prefer to check.

Reply:
It gets resolved in `lstmcpipe_start.py`, eg here:
https://github.com/cta-observatory/lstmcpipe/blob/d4f6c634772fc6a51649a8183f180814948b5656/lstmcpipe/lstmcpipe_start.py#L138